### PR TITLE
delaunator-cpp@c1521f6

### PR DESCRIPTION
**Pull request by Vertexwahn**
delaunator-cpp@c1521f6

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (delaunator-cpp) have been updated in this PR. Please review the changes.

**Comment by Vertexwahn** [comment]
Presubmit reports `BcrValidationResult.FAILED: The overlay file `BUILD.bazel` does not exist` - I assume we have to wait here for the release of Bazel 7.2.1

**Comment by Vertexwahn** [comment]
Presubmit still reports 

> BcrValidationResult.FAILED: The overlay file `BUILD.bazel` does not exist`

@fmeum @meteorcloudy  Any hints on what I am doing wrong here?

**Comment by Vertexwahn** [comment]
Waiting for https://github.com/bazelbuild/bazel-central-registry/pull/2315 to get merged

**Comment by Vertexwahn** [comment]
@fmeum Can you review this?

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
+    - windows
+  bazel:
+    - 7.2.1
```
Comment (meteorcloudy) [inline modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml:11]: nit: maybe just `7.x`?

**Review by meteorcloudy — APPROVED** [review]

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml (hunk 1 above)
Comment (Vertexwahn) [reply modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml]: Change to 7.x!

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,9 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "delaunator-cpp",
+    hdrs = ["include/delaunator.hpp"],
+    includes = ["include/"],
```
Comment (fmeum) [inline modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel:6]: Does this need both `includes` and `strip_include_prefix`? I've also never seen the trailing slash in this context, but that doesn't mean I know it's wrong.

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    - ubuntu2204
+    - windows
+  bazel:
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
```
Comment (fmeum) [inline modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml:17]: Could you add the build flag `--process_headers_in_dependencies`? It's supported by Bazel 7.2 and gives some assurance for header-only libraries.

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel (hunk 2 above)
Comment (Vertexwahn) [reply modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel]: Yes you are right. Does not make sense. Removed `strip_include_prefix`. Removed `/` from `includes`.

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml (hunk 3 above)
Comment (Vertexwahn) [reply modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml:19]: Done.

**Comment by Vertexwahn** [comment]
I applied all the suggested changes. Ready to go!

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    - windows
+  bazel:
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - '--process_headers_in_dependencies'
```
Comment (fmeum) [inline modules/delaunator-cpp/0.0.0-20181007-c1521f6/presubmit.yml:18]: Sorry, this also needs `--features=parse_headers`

**Comment by Vertexwahn** [comment]
Maybe now?

File: modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel [hunk 5]
```diff
@@ -0,0 +1,11 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "delaunator-cpp",
+    hdrs = ["include/delaunator.hpp"],
+    features = [
+        "-parse_headers",
```
Comment (fmeum) [inline modules/delaunator-cpp/0.0.0-20181007-c1521f6/overlay/BUILD.bazel:7]:
```suggestion
        "parse_headers",
```

**Review by fmeum — APPROVED** [review]

**Review by fmeum — APPROVED** [review]

**Pull request merged**